itest/starboard_cli_test.go

```diff
@@ -29,6 +29,41 @@ import (
 	. "github.com/onsi/gomega"
 )
 
+// NewPodSpec returns the creation a pod spec
+func NewPodSpec(podNamespace, podName string, containers map[string]string, secretName string) (*corev1.Pod, error) {
```

Comment:
Most of the time you pass blank string as secretName and it's 4th arg to this method. This means that either we have a method that does too many things or we can introduce maybe variadic function with option function.